packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -46,7 +46,7 @@ export class Popup extends BaseComponent<IPopupProps, {}> {
       // apply the correct focus. Without the setTimeout, we end up focusing the correct thing, and then React wants
       // to reset the focus back to the thing it thinks should have been focused.
       setTimeout(() => {
-        if (this._originalFocusedElement) {
+        if (this._originalFocusedElement && getDocument().activeElement === getDocument().body) {
```

Comment:
if without this change, we cannot do nested callout, then check it in.  Otherwise we can hold it since the timing issue can be resolve my asking caller to sequencing its callout ordering to ensure focus back set correctly.